messaging/src/main/java/org/axonframework/eventhandling/deadletter/jpa/JpaSequencedDeadLetterQueue.java

```diff
@@ -0,0 +1,716 @@
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> entityManagerProvider(
+                EntityManagerProvider entityManagerProvider) {
+            assertNonNull(entityManagerProvider, "EntityManagerProvider may not be null");
+            this.entityManagerProvider = entityManagerProvider;
+            return this;
+        }
+
+        /**
+         * Sets the {@link TransactionManager} used to manage transaction around fetching event data. Required by
+         * certain databases for reading blob data.
+         *
+         * @param transactionManager a {@link TransactionManager} used to manage transaction around fetching event data
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> transactionManager(TransactionManager transactionManager) {
+            assertNonNull(transactionManager, "TransactionManager may not be null");
+            this.transactionManager = transactionManager;
+            return this;
+        }
+
+        /**
+         * Sets the {@link Serializer} to deserialize the events, metadata and diagnostics of the {@link DeadLetter}
+         * when storing it to a database.
+         *
+         * @param serializer The serializer to use
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> eventSerializer(Serializer serializer) {
+            assertNonNull(serializer, "The serializer may not be null");
+            this.serializer = serializer;
+            return this;
+        }
+
+        /**
+         * Removes all current converters currently configured, including the default
+         * {@link EventMessageDeadLetterJpaConverter}.
+         *
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> clearConverters() {
+            this.converters.clear();
+            return this;
+        }
+
+        /**
+         * Adds a {@link DeadLetterJpaConverter} to the configuration, which is used to deserialize items entries from
+         * the database.
+         *
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> addConverter(DeadLetterJpaConverter<EventMessage<?>> converter) {
+            this.converters.add(converter);
+            return this;
+        }
+
+        /**
+         * Sets the processing group, which is used for storing and quering which event processor the deadlettered item
+         * belonged to.
+         *
+         * @param processingGroup The processing group of this {@link SequencedDeadLetterQueue}.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> processingGroup(String processingGroup) {
```

Comment:
I would expect validation of the provided `processingGroup` in this method through the `BuilderUtils`.